src/pages/test-report/__tests__/test-report.analytics.effects.spec.ts

```diff
@@ -347,4 +348,22 @@ describe('Test Report Analytics Effects', () => {
     });
   });
 
-});
+  describe('testTermination', () => {
+    it('should call logEvent for the termination event', () => {
+      // ARRANGE
+      spyOn(analyticsProviderMock, 'logEvent').and.callThrough();
+      store$.dispatch(new journalActions.StartTest(123456));
+      // ACT
+      actions.next(new testReportActions.TerminateTestFromTestReport());
+      // ASSERT
+      effects.testTermination$.subscribe((result) => {
+        expect(analyticsProviderMock.logEvent).toHaveBeenCalledWith(
+          AnalyticsEventCategories.TERMINATION,
```

Comment:
you need a callback in this else it just skips the async stuff

Reply:
Good spot. Fixed.